api/src/main/java/net/kyori/adventure/text/object/PlayerHeadObjectContents.java

```diff
@@ -51,6 +51,13 @@
  * @sinceMinecraft 1.21.9
  */
 public interface PlayerHeadObjectContents extends ObjectContents {
+  /**
+   * The default value for whether the player's hat layer should render.
+   *
+   * @since 4.25.0
+   */
+  boolean HAT_DEFAULT = true;
```

Comment:
just so it's more in keeping with other areas of the code (e.g. ClickCallback.DEFAULT_LIFETIME)
```suggestion
  boolean DEFAULT_HAT = true;
```

Reply:
Fixed it